tests/integration/compose/docker_compose_iceberg_lakekeeper_catalog.yml

```diff
@@ -0,0 +1,119 @@
+services:
+  lakekeeper:
+    image: quay.io/lakekeeper/catalog:latest-main
+    environment:
+      - LAKEKEEPER__PG_ENCRYPTION_KEY=This-is-NOT-Secure!
+      - LAKEKEEPER__PG_DATABASE_URL_READ=postgresql://postgres:postgres@db:5432/postgres
+      - LAKEKEEPER__PG_DATABASE_URL_WRITE=postgresql://postgres:postgres@db:5432/postgres
+      - RUST_LOG=info
+      - LAKEKEEPER__AUTH__DISABLED=true
```

Comment:
This flag does not exist and I think could be confusing for users that might just try to set it to `false` when they want some kind of Authentication.
Authentication in Lakekeeper is off if neither `LAKEKEEPER__ENABLE_KUBERNETES_AUTHENTICATION` nor `LAKEKEEPER__OPENID_PROVIDER_URI` is set.

```suggestion
```